### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by michael-rhoese** [janishar/JPost#6]
@janishar now it is years ago since the last release of this project was pubilshed. Do you think that the project comes back to life?

**Comment by janishar** [comment]
@michael-rhoese The project is still valid for usecase but for Android, we can go with RxBus as the alternative.

**Comment by michael-rhoese** [comment]
Of course, there are alternatives for Android based apps. But when I look at the Java libraries that support reactive programming with Eventbus capability, there are only large libraries. II think this library has the potential to be a small library that works with reactive frameworks like springs reactor. Yes, I can build a semi "eventbus" with reactor but there is no support of CbA (I call it Configuration by Annotation) and that makes it feel wrong.
